starters/next-wordpress-starter/next.config.js

```diff
@@ -38,6 +38,12 @@ if (process.env.WPGRAPHQL_URL === undefined) {
 // remove trailing slash if it exists
 imageDomain = imageDomain.replace(/\/$/, '');
 
+if (process.env.PANTHEON_ENVIRONMENT_URL) {
+	if (process.env.PANTHEON_ENVIRONMENT_URL.startsWith('live') === true) {
+		process.env.IS_LIVE_ENVIRONMENT = 'live';
```

Comment:
Technically any string will work here, but let's set it to 'true' for consistency with the docs